unittests/test_distribute_tests.py

```diff
@@ -15,38 +15,40 @@
 def default_exec_ctx(make_exec_ctx_g):
     yield from make_exec_ctx_g(system='sys0')
 
+
 @pytest.fixture
 def loader():
     return RegressionCheckLoader([
         'unittests/resources/checks_unlisted/alloc_check.py'
     ])
 
 
-def test_dynamic_testcases(loader, default_exec_ctx):
+def test_distribute_testcases(loader, default_exec_ctx):
     testcases = executors.generate_testcases(loader.load_all())
     testcases = filter(
-        filters.have_name('Simple'), testcases
+        filters.have_any_name('Simple'), testcases
     )
 
     testcases = list(testcases)
     assert len(testcases) == 4
-    count = sum(map(lambda x : x._partition.fullname == 'sys0:p0', testcases))
+    count = sum(map(lambda x: x._partition.fullname == 'sys0:p0', testcases))
     assert count == 2
-    count = sum(map(lambda x : x._partition.fullname == 'sys0:p1', testcases))
+    count = sum(map(lambda x: x._partition.fullname == 'sys0:p1', testcases))
     assert count == 2
 
     node_map = {
         'sys0:p0': ['n1', 'n2'],
         'sys0:p1': ['n3']
     }
-    new_cases = distribute_tests(testcases, False, False, node_map)
+    new_cases = distribute_tests(testcases, node_map)
     assert len(new_cases) == 6
-    count = sum(map(lambda x : x._partition.fullname == 'sys0:p0', new_cases))
+    count = sum(map(lambda x: x._partition.fullname == 'sys0:p0', new_cases))
```

Comment:
Don't access private fields here and generally:

```suggestion
    count = sum(map(lambda x: x[1].fullname == 'sys0:p0', new_cases))
```